src/components/docs/Docs/Sidebar/Sidebar.module.scss

```diff
@@ -91,7 +97,7 @@
 }
 
 .sidebartitle {
-    margin: 18px 0 0 2px;
+    margin: 24px 0 0 2px;
     padding-bottom: 10px;
```

Comment:
just a small question, we don’t use 'rem' on this project?

Reply:
Oh, that is a good point. I don't think rem has been used in this project before, but it can be changed to be more conform with other projects. I'll make a start by converting these to rem. Over time we can change more to rem values.

Reply:
@hubert-deriv please revert your `px to rem` commit.
`rem` is relative to the root element font size and in this project the root font-size is `16px`/`1rem`.

To avoid confusing better to stick to `px` for now and we can have a separate improvement card to update the base font size to `10px` and use a respective `rem` values which is more straightforward and also follow the same standard as Deriv-app project.

Ref: https://www.sitepoint.com/understanding-and-using-rem-units-in-css/

<img width="262" alt="Screenshot 2022-10-26 at 12 29 04 PM" src="https://user-images.githubusercontent.com/70938039/197934739-f3950755-2dd3-4d39-b731-75b8bb91e561.png">

Reply:
yes, I should consider it while leaving a comment here.. I'm sorry, Hubert

Reply:
No worries, simple git revert and it's done 👍

Reply:
I will make a seperate improvement as @matin-binary suggested